common/config/config.go

```diff
@@ -35,6 +35,16 @@ import (
 
 var Version = "" //Set value when build project
 
+type VerifyMethod int
+
+const (
+	NoneVerifyMethod   = VerifyMethod(0)
+	InterpVerifyMethod = VerifyMethod(1)
+	JitVerifyMethod    = VerifyMethod(2)
+)
+
+var WasmVerifyMethod = InterpVerifyMethod
```

Comment:
this should be removed. the default is already handled by `WasmVerifyMethodFlag`